packages/opal-server/opal_server/scopes/loader.py

```diff
@@ -0,0 +1,35 @@
+from opal_common.schemas.policy_source import GitPolicySource, NoAuthData, SSHAuthData
+from opal_common.schemas.scopes import Scope
+from opal_server.config import ServerRole, opal_server_config
+from opal_server.scopes.scope_repository import ScopeRepository
+
+
+async def load_scopes(repo: ScopeRepository):
+    if opal_server_config.SERVER_ROLE == ServerRole.Leader:
+        await _load_env_scope(repo)
+
+
+async def _load_env_scope(repo: ScopeRepository):
+    # backwards compatible opal scope
+    if opal_server_config.POLICY_REPO_URL is not None:
+        auth = NoAuthData()
+
+        scope = Scope(
+            scope_id="env",
```

Comment:
nit: not sure "env" is the correct term here. might confuse a future developer.
maybe "default" scope?